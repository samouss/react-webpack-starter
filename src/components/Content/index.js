import React from 'react';
import PropTypes from 'prop-types';
import './index.css';

const Header = ({ message }) => (
  <p styleName="container">
    Hello from
    {message}
    !!
  </p>
);

Header.propTypes = {
  message: PropTypes.string.isRequired,
};

export default Header;
