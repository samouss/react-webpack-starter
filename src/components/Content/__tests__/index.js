import React from 'react';
import { shallow } from 'enzyme';
import Content from '../index';

describe('<Content />', () => {
  it('should render the component', () => {
    const props = {
      message: 'Test app',
    };

    const component = shallow(
      <Content
        {...props}
      />,
    );

    expect(component).toMatchSnapshot();
  });
});
