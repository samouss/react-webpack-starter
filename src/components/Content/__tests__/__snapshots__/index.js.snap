// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Content /> should render the component 1`] = `
<p
  styleName="container"
>
  Hello from
  Test app
  !!
</p>
`;
